src/bundle/Resources/public/js/scripts/button.state.checkbox.toggle.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc) {
+    const toggleForms = [...doc.querySelectorAll('.ez-toggle-btn-state-checkbox')];
+    const ALL_CHECKED = 'all-checked';
+    const ANY_CHECKED = 'any-checked';
+
+    toggleForms.forEach((toggleForm) => {
+        const checkboxInputs = [...toggleForm.querySelectorAll('input[type="checkbox"]')];
+        const toggleButtonState = () => {
```

Comment:
This function should be implemented outside of `forEach` loop. If you want to have an access to the `checkboxInputs` then a useful piece of information is that a `forEach` loop callback takes 3 params: item, index, an iterated array with all values. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach